core/components/atoms/alert/alert.js

```diff
@@ -119,7 +119,12 @@ Alert.Element = styled.div`
     right: 0;
     top: 0;
     color: ${props => colors.alert[props.type].text};
+    opacity: .2;
     padding: ${spacing.small} ${spacing.small};
+    &:hover {
+      color: rgba(0,0,0);
```

Comment:
Can't this just be `rgba(0, 0, 0, .5)` instead? and remove the `opacity` propierty?

@francocorreasosa shouldn't a color declaration be a token?

Reply:
Yeah you're right, I should probably have imported the black color token and used that instead. In which case it would be:
`
&:hover {
      color: black;
      opacity: .5;
`

I think the opacity line is needed because I set the regular state to `opacity .2`, which will cascade down unless you change it in the hover state. I didn't know of any other way to set opacity on a token.

Reply:
@mrewers I wouldn't make it black, just keep the original color and change the opacity.

```css
&:hover { opacity: .5; }
```

@francocorreasosa, not related to this PR, but for later refactoring I think opacities values should be tokens.

Reply:
@andresgalante Will do. Thanks.